framework/src/modules/chain/transport.js

```diff
@@ -231,74 +229,75 @@ class Transport {
 			 * @todo Add @returns tag
 			 * @todo Add description of the function
 			 */
-			blocksCommon(query, cb) {
+			async blocksCommon(query) {
 				query = query || {};
-				return library.schema.validate(
+
+				const valid = library.schema.validate(
```

Comment:
isn't this async?

Reply:
No. It's the sync mode of `z-schema`

REF.: https://github.com/zaggino/z-schema#sync-mode